aem65/src/main/java/com/cognifide/qa/bb/aem65/tests/GuiceModule.java

```diff
@@ -18,5 +20,6 @@
   @Override
   protected void configure() {
     bind(TextComponent.class).to(TextComponentImpl.class);
+    bind(TeaserComponent.class).to(TeaserComponentImpl.class);
```

Comment:
There's no need to create the TeaserComponent interface. I recommend deleting the interface and renaming the TeaserComponentImpl class to TeaserComponent. No binding in the Guice Module required then

Reply:
done